packages/atomic-react/src/components/stencil-generated/commerce/index.ts

```diff
@@ -36,7 +36,9 @@ export const AtomicCommerceSortDropdown = /*@__PURE__*/createReactComponent<JSX.
 export const AtomicCommerceText = /*@__PURE__*/createReactComponent<JSX.AtomicCommerceText, HTMLAtomicCommerceTextElement>('atomic-commerce-text');
 export const AtomicCommerceTimeframeFacet = /*@__PURE__*/createReactComponent<JSX.AtomicCommerceTimeframeFacet, HTMLAtomicCommerceTimeframeFacetElement>('atomic-commerce-timeframe-facet');
 export const AtomicIcon = /*@__PURE__*/createReactComponent<JSX.AtomicIcon, HTMLAtomicIconElement>('atomic-icon');
+export const AtomicInsightUserActionsModal = /*@__PURE__*/createReactComponent<JSX.AtomicInsightUserActionsModal, HTMLAtomicInsightUserActionsModalElement>('atomic-insight-user-actions-modal');
 export const AtomicInsightUserActionsTimeline = /*@__PURE__*/createReactComponent<JSX.AtomicInsightUserActionsTimeline, HTMLAtomicInsightUserActionsTimelineElement>('atomic-insight-user-actions-timeline');
+export const AtomicInsightUserActionsToggle = /*@__PURE__*/createReactComponent<JSX.AtomicInsightUserActionsToggle, HTMLAtomicInsightUserActionsToggleElement>('atomic-insight-user-actions-toggle');
```

Comment:
The reason why I didn't mark them as internal, is because when I did, the public properties of these components were not made interactive in Storybook so I was not able to test these components:

for example, with `@internal` that's how storybook renders:
<img width="1726" alt="Screenshot 2024-09-12 at 11 31 51 AM" src="https://github.com/user-attachments/assets/b5787053-bb4a-4a06-a5f9-93c2c389e6cc">

no inputs to introduce the `ticketCreationDateTime` or the `userId` parameters.

Reply:
I tried tagging the component with @insight, but they still appear in generated files :/ any idea how to surpass this?

Reply:
solved!